src/main/java/org/opentripplanner/street/model/vertex/Vertex.java

```diff
@@ -198,6 +199,26 @@ public List<StreetEdge> getOutgoingStreetEdges() {
     return result;
   }
 
+  public boolean traversalBanned(State currentState) {
+    return rentalRestrictions.traversalBanned(currentState);
+  }
+
+  public void addRentalRestriction(RentalRestrictionExtension ext) {
+    rentalRestrictions = rentalRestrictions.add(ext);
+  }
+
+  public RentalRestrictionExtension rentalRestrictions() {
+    return rentalRestrictions;
+  }
+
+  public boolean dropOffBanned(State currentState) {
```

Comment:
Like `rentalDropOffBanned `

Reply:
BTW, how does it work if the network has stations, can you drop off the vehicle at those inside a no-drop-off-zone?

Reply:
As a simplification we only check if the network has restrictions on that vertex so it would not be allowed to drop it off if rented from a station. (If it does then that would be a bug.)

The GBFS standard allows you to specify different zones depending on the vehicle type but I have not implemented that.

Reply:
Done in 3486a8308b5a00374f1bc9afa0c1713870a76642